README.md

```diff
@@ -8,8 +8,27 @@ it in ocp environment. After an operator is installed a pre-flight tests are exe
 that validates that operator meets minimum requirements for Red Hat OpenShift Certification.
 If tests pass a CI pipeline submits a PR for full operator certification workflow.
 
-To trigger a CI pipeline follow steps bellow:
 
+Since CI pipeline needs to make some changes in git repository (for example digest pinning)
```

Comment:
Perhaps a better example is for opening a pull request at the end.

Reply:
SSH key won't be used for the opening pull requests. An SSH key is only used to push changes to user's repo and GitHub API key will be used for pull requests.